src/main/kotlin/cc/hyperium/Hyperium.kt

```diff
@@ -35,6 +36,7 @@ object Hyperium {
     )
     private val logger: Logger = LogManager.getLogger()
     private val runningServices = ServiceRegistry()
+    private val resourceManager = ResourceManager()
```

Comment:
We don't really want to just have a bunch of fields in our Hyperium object, that gets weird and unruly very fast. I would prefer to make this provided through kodein as a singleton. Or, if you are going to make ResourceManager a service as I suggested below, it will automatically be accessible through Kodein.